index.markdown

```diff
@@ -621,6 +621,11 @@ Capital One credit card that's directly targeted for people with no credit
 history/newcomers. Beware this card is free for the first year, but costs around
 $30 per year after that.
 
+#### Wells Fargo
+
+Stefan got declined twice trying to get a secured credit card there. Both times they 
+said my SSN was invalid. If you want a CC, wouldn't recommend Wells Fargo.
```

Comment:
That's to be expected right? You wont have credit history until you get a line of credit.

Generally the approach for someone new to the US is to go to their bank and get a secured credit card. Or if you've had a checking account for some time you can sometimes bring in your offer letter and get a regular card. Some times (discussed in the guide) you can even get them to look at your Canadian history.